src/components/grid/grid.vue

```diff
@@ -221,6 +250,9 @@
       this.$forceUpdate();
     },
     methods: {
+      isRenderer(column) {
+        return column.render && column.render.use;
```

Comment:
```suggestion
        return column && column.render && column.render.use;
```

정확하게 이 메소드에 어떤 값이 들어갈 것인지 `isRenderer`  만 봐서는 알 수가 없으므로 column 부터 검사하는게 안정성이 더 좋을 것 같습니다. (느낌상 152번 column 이 들어올 것 같긴하지만)

Reply:
네 column은 무조건 존재할수밖에 없는 데이터라 조건에선 제외를 했는데 명시적인게 좋을까요?

Reply:
아니면 가장 간단하게는
```js
isRenderer(column = {}) {
  return column.render && column.render.use;
}
```

Reply:
애길들어보니 제가 만들어서 확신하는거지 팀원들은 확신할수 없겠네요.  
추가해야겠어요